Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/Helpers/Utils.cs

```diff
@@ -45,6 +45,39 @@ public static async Task<Relationship> EstablishRelationshipBetween(Client clien
         return getRelationshipResponse.Result!;
     }
 
+    public static async Task<Relationship> CreateRejectedRelationshipBetween(Client client1, Client client2)
```

Comment:
You should extract common code from this method and `EstablishRelationshipBetween`. The extracted method could be called `CreatePendingRelationshipBetween`

Reply:
done